src/components.tsx

```diff
@@ -299,6 +299,10 @@ const LogLineRaw = ({
       topClasses.push("normal-line");
       content = InstructionLineContent({ line, frame });
       break;
+    case ParsedLineType.C_SOURCE:
+      topClasses.push("inline-c-source-line");
```

Comment:
Do we need the "inline" prefix to differentiate between this class and what's going to go in the c source view? (this is a nit)